main.go

```diff
@@ -55,7 +55,6 @@ var (
 func init() {
 	utilruntime.Must(clientgoscheme.AddToScheme(scheme))
 	utilruntime.Must(applicationapiv1alpha1.AddToScheme(scheme))
-	utilruntime.Must(integrationv1alpha1.AddToScheme(scheme))
```

Comment:
I'm not sure what's the purpose of this, but don we have to add v1beta1 there instead?

Reply:
I think that beta is already added [here](https://github.com/redhat-appstudio/integration-service/blob/e8610588997cfaf49bbcf66d224c90964ffa10c3/main.go#L59).

Reply:
ofc, I need glasses